eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/JournalHead.scala

```diff
@@ -6,7 +6,13 @@ import com.evolutiongaming.kafka.journal.{DeleteTo, PartitionOffset, SeqNr}
 import com.evolutiongaming.scassandra.syntax._
 import com.evolutiongaming.scassandra.{DecodeRow, EncodeRow}
 
-
+/** Represent metadata of the latest journal event saved to Cassandra.
+  *
+  * @param partitionOffset the offset where the event is located in Kafka
+  * @param segmentSize the last segmentSize used to write a journal
```

Comment:
it's not last, it's constant for life of journal

Reply:
What is expected to happen when we switch to larger `segmentSize`? Would it only be used for new journals?